src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Values/HostingEnvironmentValue.cs

```diff
@@ -0,0 +1,54 @@
+using System.Globalization;
+using System.Text.Encodings.Web;
+using Fluid;
+using Fluid.Values;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace OrchardCore.DisplayManagement.Liquid.Values;
+
+internal sealed class HostingEnvironmentValue : FluidValue
+{
+    private readonly IHostEnvironment _hostEnvironment;
+
+    public HostingEnvironmentValue(IHostEnvironment hostEnvironment = null)
+        => _hostEnvironment = hostEnvironment;
+
+    public override FluidValues Type => FluidValues.Object;
+
+    public override bool Equals(FluidValue other)
+    {
+        if (other is null)
+        {
+            return false;
+        }
+
+        return ToStringValue() == other.ToStringValue();
+    }
+
+    public override bool ToBooleanValue() => false;
```

Comment:
```suggestion
    public override bool ToBooleanValue() => true;
```

An existing object should be truthy. Same for `CultureValue`.

This way you can do `{% if Environment %}` to test if the object exists.